internal/core/model.rs

```diff
@@ -175,11 +175,57 @@ pub trait Model {
     /// assert_eq!(handle.row_data(3).unwrap(), 4);
     /// ```
     ///
-    /// Note: the default implementation returns nothing interesting. this method should be
-    /// implemented by model implementation to return something useful. For example:
+    /// Note: Custom models must implement this method for the cast to succeed.
+    /// A valid implementation is to return `self`:
     /// ```ignore
     ///     fn as_any(&self) -> &dyn core::any::Any { self }
     /// ```
+    ///
```

Comment:
```suggestion
    ///
    /// ## Troubleshooting
```

Just to make clear that this is not reference material.

Reply:
Good idea!